portal/src/main/webapp/WEB-INF/jsp/dashboard/dashboard.jsp

```diff
@@ -569,16 +570,9 @@
     
     
     </span>
-    
-    
-<script src="js/src/load-frontend.js"></script>
 
-<script>
-        window.loadReactApp({ defaultRoute: 'blank' });
-</script>
 
-<div id="reactRoot" style="display:none"></div>
-    
     
+
```

Comment:
@kalletlak @zhx828 getting rid of import of new app into old studyview jsp.  was causing some complications